SampleApp/SampleApp.xcodeproj/project.pbxproj

```diff
@@ -754,7 +754,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Note - we could eliminate the need to maintain this with a post install action (thereby reducing the number of sources of truth), but it seemed unnecessary to introduce as much in this PR.